scripts/lint_allowed_geth_imports.sh

```diff
@@ -8,7 +8,7 @@ set -o pipefail
 # 1. Recursively search through all go files for any lines that include a direct import from go-ethereum
 # 2. Sort the unique results
 # #. Print out the difference between the search results and the list of specified allowed package imports from geth.
-extra_imports=$(grep -r --include='*.go' --exclude-dir='simulator' '"github.com/ethereum/go-ethereum/.*"' -o -h | sort -u | comm -23 - ./scripts/geth-allowed-packages.txt)
+extra_imports=$(grep -Pr --include='*.go' --exclude-dir='simulator' '^\s+"github.com/ethereum/go-ethereum/.*"' -o -h | tr -d '[:blank:]' | sort -u | comm -23 - ./scripts/geth-allowed-packages.txt)
```

Comment:
There was a false positive on `const geth = "github.com/ethereum/go-ethereum/"` so I updated the check to only match with (possible) whitespace but no other prefix.